pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -313,6 +317,17 @@ public BrokerService(PulsarService pulsar) throws Exception {
         this.backlogQuotaManager = new BacklogQuotaManager(pulsar);
         this.backlogQuotaChecker = Executors
                 .newSingleThreadScheduledExecutor(new DefaultThreadFactory("pulsar-backlog-quota-checker"));
+        this.pulsar.addPrometheusRawMetricsProvider(stream -> {
```

Comment:
From the broker side, I think you can generate the metrics directly. The RawMetricsProvider is used by some plugins who want to expose metrics through the /metrics endpoint.

Reply:
I think you can follow the current broker metrics(topics metrics, namespace metrics) generate way.

Reply:
Thanks.